CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # Changelog
 
+## 0.0.3 - 2023-05-04
+
+### Features
+
+- **Individual Changelog pages: [shadows-withal]/[↬284]**
+    
+    When announcing a new release- it's often desirable to link to an individual
+    release page that contains the changelog/release notes. Previously, we built
+    a single page for all the releases- now we build individual pages as well.
+
+    This is the first shipped feature from our new team member, Liv! Yay and 
+    welcome :)
+
+### Fixes
+
+- **Improved configuration support for non-cargo dist users - [pomdtr]/[#262] , [ashleygwilliams]/[↬281]**
+
+    Previously, setting `cargo-dist` as false, or omitting it should have been
+    sufficient to stop oranda from attempting to parse your releases as
+    cargo-dist artifacts, however `cargo-dist: false` did not work! How the
+    entire artifacts config object is handled has been reworked and tested.
+    Addiitionally, work from a refactor has also made project with a mixture of
+    cargo-dist and non-cargo-dist releases work much better.
+
+- **Installer header fallback when platform is not detected: [gankra]/[↬269]**
+
+    When you use cargo-dist, we display a header on your index page that
+    detects your user's platform and recommends and installer to them. We did
+    not previously have a fallback if we detected a system that none of the
+    installers supported. Now, in that scenario- we'll offer an artifact
+    download as the header option.
+
+- **Dev commands has proper default values: [ashleygwilliams]/[#256],[↬260]**
+
+    Due to a false hope that `#[derive(Default)]` would collect default from 
+    the `clap` derive API, we shipped the `dev` command with null defaults.
```

Comment:
I would reword the end of this, maybe "with no defaults".

Reply:
ah- so there are defaults ... like it "works" but if you derive default on a struct it functionally sets everything to the "null" option (0, none, etc)

Reply:
hm, but then i struggle to see how this contributes to the end user experience here... i still think "null defaults" sounds kinda weird. is "nulled" a real word we can use here?

Reply:
or actually maybe just put `null` in backticks so people know it's a computer sciencey term instead of the uh, number i guess

Reply:
OH i guess that's why it's so weird to me - null means zero in german lol

Reply:
i can update this to be more specific!!